test/e2e/connectivity_test.go

```diff
@@ -31,6 +31,67 @@ import (
 
 const pingCount = 5
 
+// TestAllConnectivity is the top-level test which contains all subtests for
+// Connectivity related test cases so they can share setup, teardown.
+func TestAllConnectivity(t *testing.T) {
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	t.Run("TestAllConnectivity", func(t *testing.T) {
+		t.Run("testPodConnectivityOnSameNode", func(t *testing.T) { testPodConnectivityOnSameNode(t, data) })
+		t.Run("testHostPortPodConnectivity", func(t *testing.T) {
+			skipIfHasWindowsNodes(t)
+			testHostPortPodConnectivity(t, data)
+		})
+		t.Run("testPodConnectivityDifferentNodes", func(t *testing.T) {
+			skipIfNumNodesLessThan(t, 2)
+			testPodConnectivityDifferentNodes(t, data)
+		})
+		t.Run("testOVSRestartSameNode", func(t *testing.T) {
+			skipIfProviderIs(t, "kind", "test not valid for the netdev datapath type")
+			skipIfNotIPv4Cluster(t)
+			skipIfHasWindowsNodes(t)
+			testOVSRestartSameNode(t, data)
+		})
+		t.Run("testOVSFlowReplay", func(t *testing.T) {
+			skipIfProviderIs(t, "kind", "stopping OVS daemons create connectivity issues")
+			skipIfHasWindowsNodes(t)
+			testOVSFlowReplay(t, data)
+		})
+		t.Run("testPingLargeMTU", func(t *testing.T) {
+			skipIfNumNodesLessThan(t, 2)
+			testPingLargeMTU(t, data)
+		})
+	})
+}
+
+// TestPodConnectivityAfterAntreaRestart checks that restarting antrea-agent does not create
+// connectivity issues between Pods.
+// Notes: TestPodConnectivityAfterAntreaRestart will fail if it's added in TestAllConnectivity
+// to share the setup, teardown with other cases due to it redeploy Antrea during test.
+func TestPodConnectivityAfterAntreaRestart(t *testing.T) {
```

Comment:
Can you move it back to its original location so it won't appear as a diff (saving review effort)?